pkg/logs/internal/tailers/file/tailer.go

```diff
@@ -157,15 +169,38 @@ func NewTailer(outputChan chan *message.Message, file *File, sleepDuration time.
 		stopForward:            stopForward,
 		isFinished:             atomic.NewBool(false),
 		didFileRotate:          atomic.NewBool(false),
-		info:                   info,
+		info:                   opts.Info,
 		bytesRead:              bytesRead,
 	}
+
+	if fileRotated {
+		addToTailerInfo("File Rotated", strconv.FormatBool(fileRotated), t.info)
+		addToTailerInfo("Last Rotation Date", log.GetFormattedTime(), t.info)
+	}
+
+	return t
+}
+
+// addToTailerInfo add NewMappedInfo with a key value pair into the tailer info for displaying
+func addToTailerInfo(k, m string, tailerInfo *status.InfoRegistry) {
+	newInfo := status.NewMappedInfo(k)
+	newInfo.SetMessage(k+":", m)
```

Comment:
Isn't the `:`[ automatically aded here?](https://github.com/DataDog/datadog-agent/blob/65a17c4c88968f01ef092312b5ba9a555496ad61/pkg/status/templates/logsagent.tmpl#L103)